### PR TITLE
Premint: add creator to creator attribution event

### PR DESCRIPTION
**Pull request by oveddan**
* Add creator address to creator attribution event.
* Convert creator attribution fields for `name` and `version` from `bytes32` to `string` (to follow the standard)
* Added test that verifies creator attribution event emitted

### CODE REVIEW

**Comment by oveddan** [comment]
Current dependencies on/for this PR:
* main
  * **PR #161** <a href="https://app.graphite.dev/github/pr/ourzora/zora-1155-contracts/161" target="_blank">Graphite</a> 
    * **PR #163** <a href="https://app.graphite.dev/github/pr/ourzora/zora-1155-contracts/163" target="_blank">Graphite</a>  👈
    * **PR #162** <a href="https://app.graphite.dev/github/pr/ourzora/zora-1155-contracts/162" target="_blank">Graphite</a> 

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/ourzora/zora-1155-contracts/163?utm_source=stack-comment).

File: src/nft/ZoraCreator1155Impl.sol [hunk 1]
```diff
@@ -727,10 +727,6 @@ contract ZoraCreator1155Impl is
     /* start eip712 functionality */
     mapping(uint32 => uint256) public delegatedTokenId;
```
Comment (kulkarohan) [inline src/nft/ZoraCreator1155Impl.sol:728]: should we put this in a `PremintStorage` contract?

**Comment by kulkarohan** [comment]
other than that ^ looks good to merge

File: src/nft/ZoraCreator1155Impl.sol (hunk 1 above)
Comment (oveddan) [reply src/nft/ZoraCreator1155Impl.sol:728]: yes to be consistent that makes sense. It's out of the scope of this PR, but can do that as a separate PR.

Comment (kulkarohan) [reply src/nft/ZoraCreator1155Impl.sol:728]: sounds good

**Review by kulkarohan — APPROVED** [review]

**Pull request merged**